src/main/java/org/reaktivity/nukleus/kafka/internal/cache/PartitionIndex.java

```diff
@@ -0,0 +1,37 @@
+package org.reaktivity.nukleus.kafka.internal.cache;
+
+import java.util.Iterator;
+
+import org.agrona.DirectBuffer;
+import org.reaktivity.nukleus.kafka.internal.stream.HeadersFW;
+import org.reaktivity.nukleus.kafka.internal.types.OctetsFW;
+
+public interface IPartitionIndex
```

Comment:
We don't follow the `I` prefix to indicate `interface` convention, please remove leading `I`.

Reply:
yes, intermediate state, fixed now